### PR TITLE
#2525 Add a generator model with RPCL for DynaFlow

### PR DESCRIPTION
**Pull request by cossartque**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

**Review by rosiereflo — APPROVED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Logs/package.order [hunk 1]
```diff
@@ -1,2 +1,6 @@
 Timeline
 Constraint
+ErrorKeys
```
Comment (marcochiaramello) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Logs/package.order:3]: why this file has changed?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo [hunk 2]
```diff
@@ -22,9 +22,14 @@ model GeneratorPV "Model for generator PV based on SignalN for the frequency han
 
   parameter Types.ReactivePowerPu QMinPu "Minimum reactive power in pu (base SnRef)";
   parameter Types.ReactivePowerPu QMaxPu "Maximum reactive power in pu (base SnRef)";
+  parameter Types.ReactivePower QNomAlt "Nominal reactive power of the generator in Mvar";
```
Comment (marcochiaramello) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo:25]: what "Alt" stays for? thx

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo (hunk 2 above)
Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo:25]: It stands for Alternator

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Logs/package.order (hunk 1 above)
Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Logs/package.order:3]: My bad, I will fix it

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo (hunk 2 above)
Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo:25]: It stands for Alternator

Comment (marcochiaramello) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo:25]: maybe we can specify "considered on alternator side", same everywhere

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo (hunk 2 above)
Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo:25]: Agreed

**Review by marcochiaramello — APPROVED** [review]

**Pull request merged**